themes/default/content/resources/pulumi-up-2023/index.md

```diff
@@ -33,7 +33,7 @@ block_external_search_index: true
 
 # The url slug for the webinar landing page. If this is an external
 # webinar, use the external URL as the value here.
-url_slug: "/pulumi-up"
+url_slug: "https://www.pulumi.com/resources/pulumi-up-2023/"
```

Comment:
is there a reason to do that instead of https://www.pulumi.com/pulumi-up/ ?

Reply:
Nevermind, there is!

Reply:
yeah it has to do with the way the service backend is set up